tests/integration/restricted/test.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+set -ex
+
+RANDOM_CLUSTER_NAME_SUFFIX=`cat /dev/urandom | tr -cd 'a-f0-9' | head -c 7`
+
+CLUSTER_NAME="ds-test-restricted-${RANDOM_CLUSTER_NAME_SUFFIX}"
+
+cd $(dirname "${BASH_SOURCE[0]}")
+
+# set up the cluster, set up the fake user and restricted policy and then deploy the non-privileged overlay as that user
+
+gcloud container clusters create ${CLUSTER_NAME} --zone ${TEST_GCP_ZONE} --num-nodes 3 --machine-type n1-standard-16 --disk-type pd-ssd --project ${TEST_GCP_PROJECT} --labels=cost-category=build
```

Comment:
I think some more labels here would be cool, extracted from the buildkite env. Mostly interested in the user attached to the build. Could pick up things like a bot causing lots of builds / etc. Another interesting one would be if its for master vs a PR.

Additionally a label sayings this is a deploy-sourcegraph integration test would be good.

Reply:
ok, will do